internal/logging/log.go

```diff
@@ -10,10 +10,22 @@ import (
 type loggerKey string
 
 const (
-	providerSpaceRootLoggerKey loggerKey = "provider"
+	// ProviderSpaceRootLoggerKey is the loggerKey that will hold the root
+	// logger for writing logs from within provider code.
+	ProviderSpaceRootLoggerKey loggerKey = "provider"
```

Comment:
Nit: Any particular reason to include "Space" in the naming as compared to SDK? Any changes should also include the Get/Set functions. 👍

Reply:
Not in particular, just some vague anxiety about it being confusing that I don't think panned out and I never went back to correct it. I'll fix it.